docs/source/plugins.rst

```diff
@@ -73,11 +66,12 @@ This plugin allows the use of `ASTRA-toolbox
 (`10.1364/OE.24.025129 <http://dx.doi.org/10.1364/OE.24.025129>`_ 
 for forward and back projections and filter back projection reconstruction. 
 
-.. autoclass:: cil.plugins.astra.processors.FBP
+.. autoclass:: cil.plugins.astra.FBP
```

Comment:
In the plugins file I notice you have subheadings inside CCPi Regularisation but not in the others. I think it would be nice to have for tigre and astra as well

Reply:
![image](https://user-images.githubusercontent.com/60604372/146013351-52cccd5b-fcfb-4143-8536-fb76608005e6.png)